network/qubes-setup-dnat-to-ns

```diff
@@ -48,6 +48,7 @@ def get_dns_resolv_conf():
     return nameservers
 
 def get_dns_resolved():
+    dns = []
```

Comment:
This line isn't needed - there is no code path that leave dns unset and gets to the place where it's used. And there would be - better get an exception that is easy to find, than _silently_ break DNS.